src/app/home/pages/HooksPage/components/AddEditClusterModal/AddEditClusterForm.tsx

```diff
@@ -0,0 +1,387 @@
+import React, { useState } from 'react';
```

Comment:
I couldn't find any cluster actions on Hooks page. I didnt understand the usage of the Cluster Edit modals and forms in the context of hooks.

Reply:
Good catch @pranavgaikwad - removed these files.

Reply:
Ahh I see. They were pasted from the directory.

Reply:
@pranavgaikwad everything seems to be working after removing those files. Yeah i forgot to remove those after planning to use them as templates.